scripts/update.rb

```diff
@@ -0,0 +1,2 @@
+bolton = House.where(name:'House Bolton')
+bolton[motto] = 'We are evil jerks'
```

Comment:
`.where` returns an active record relation (basically an array with extra methods) so you need target the zeroth element of bolton i.e:

`bolton[0][:motto] = "We are evil jerks"`